Adding some instructions for IntelliJ and Maven (#18)
gurkenlabs/litiengine-docs: getting-started/install-jdk.md

## Installing the Java Development Kit 
First of all, you should download a Java development kit (**JDK**), containing the Java virtual machine (**JVM**) and other resources for developing Java applications.
Since LITIENGINE has been created with Java 8, you need at least JDK 8 on your machine.

> > If you're on linux or mac, I would hightly recommend to use [SKDMAN!](https://sdkman.io/). If you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).

Reviewer: ```suggestion
> If you're on linux or mac, we recommend using [SKDMAN!](https://sdkman.io/). If you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).
```

## Installing the Java Development Kit 
First of all, you should download a Java development kit (**JDK**), containing the Java virtual machine (**JVM**) and other resources for developing Java applications.
Since LITIENGINE has been created with Java 8, you need at least JDK 8 on your machine.

> > In case you are on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manager your JDK installation. As soon you installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).

Reviewer: ```suggestion
> In case you are on linux or mac, you can also use [SKDMAN!](https://sdkman.io/) to manager your JDK installation. Once you've installed sdk man, type `sdk install java` in your terminal. The latest recommend JDK will be installed automatically for you. It should be registered in your IDE after you restart it. For further information look up the [SDKMAN! Docs](https://sdkman.io/usage).
```
